Урок 5. Применение математической статистики для проверки гипотез в реальной жизни для популярных метрик

1)Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%. Дайте краткие рекомендации команде .Результаты: 1)Число юзеров в группах , которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550 . 2) Число юзеров в группах , которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228 3) Конверсии : conv1 = 1.05% conv2 = 1.47% .

Решение:

Для оценки статистической значимости различий между конверсиями в двух группах при α=5% используем Z-критерий для разности долей. Формула Z-критерия:

Z = (p1 - p2) / sqrt(p * (1 - p) * (1/N1 + 1/N2))

где p = (n1 * p1 + n2 * p2) / (n1 + n2)

p1 и p2 - конверсии в группах, N1 и N2 - количество юзеров в группах, которые заходили на сайт в период эксперимента, n1 и n2 - количество юзеров, которые совершили хотя бы одну покупку за период эксперимента.

In [1]:
from math import sqrt


p1 = 164 / 15550

p2 = 228 / 15550

n1 = 164

n2 = 228

N1 = 15550

N2 = N1

p = (n1 * p1 + n2 * p2) / (n1 + n2)

Z = (p1 - p2) / sqrt(p * (1 - p) * (1/N1 + 1/N2))
Z


-3.2110930329358367

Ответ:

Z-критерий при α=5% и двухсторонней альтернативе равен 1.96.  
Расчетное значение Z меньше критического, т.е. -3.2110930329358367 < 1.96 , то мы отвергаем нулевую гипотезу о равенстве конверсий в группах.  
Значит, изменения внесенные в заказную форму дали статистически значимый эффект на уровень конверсии в покупку. Рекомендуется использовать новую форму заказа.

---

Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе . Какой статистический критерий тут лучше всего подойдёт и почему ?

Ответ:

Если размер выборки достаточно большой (более 30 элементов в каждой группе), то может быть использован z-критерий. В данном случае размер выборки составляет 10 000 элементов, то нужно использовать z-критерий.